### ISSUE TITLE
Online (web) invoice template editor

### ISSUE THREAD

**Issue by ririko5834** [InvoicePlane/InvoicePlane#760]
It would be nice to add a template editor for online (web public) invoice templates. So you can build your own invoice template (using some HTML editor) with template tags (placeholders) without the need of coding PHP skills.

**Comment by clockwiseq** [comment]
This is an enhancement that I recommended a couple of years ago.  Not sure what direction we go in with the v1 vs v2 _(branch the code and start over)_ or if we continue on with the current code base and enhance it.  Either way, this is one of my favorite requests as I have always felt that it would increase the user-base because business owners wouldn't have to depend on a PHP developer to create their own invoice templates.

**Comment by ririko5834** [comment]
The same thing should exist be for PDF invoice templates.

With this, you could change the appearance of the templates, for example, make that custom client fields will be displayed in the templates

**Comment by clockwiseq** [comment]
Yeah, that was what I meant above.  The template editor would allow you to modify and store the templates in the database and have a visual editor for ease-of-use.  Those templates would be for the three (3) areas an invoice was displayed _(web, email, PDF)_

**Comment by OPGRBILLINGDEV** [comment]
Is underlyning technology selected for editor ? What about pure CSS ?
